### PR TITLE
Patch 3

### PR DESCRIPTION
**Pull request by simomac**
adding missing publisher-place to book ref.

### CODE REVIEW

**Comment by adam3smith** [comment]
(test failure is unrelated to your style)

**Comment by simomac** [comment]
'(test failure is unrelated to your style)...' Hi, I'm fairly new to git can you advise what I need to do?

**Comment by adam3smith** [comment]
Nothing for now -- we've updated our automated checks but the branch from which you made your pull request predated those changes, so they fail.

File: harvard-imperial-college-london.csl [hunk 1]
```diff
@@ -17,9 +17,9 @@
     <updated>2021-09-28T14:37:15+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
-  <macro name="editor">
-    <names variable="editor" delimiter=", ">
-      <name and="symbol" initialize-with="."/>
+   <macro name="editor">
```
Comment (bwiernik) [inline harvard-imperial-college-london.csl:20]:
```suggestion
  <macro name="editor">
```

File: harvard-imperial-college-london.csl [hunk 2]
```diff
@@ -90,6 +91,7 @@
       <choose>
         <if type="report" match="none">
           <text variable="publisher"/>
+          <text variable="publisher-place"/>
```
Comment (bwiernik) [inline harvard-imperial-college-london.csl:94]: There is no delimiter between publisher and publisher-place here. You should put these inside a `<group delimiter=", ">`

**Review by bwiernik — CHANGES_REQUESTED** [review]

**Comment by retorquere** [comment]
The current test failure in this PR is not related to the re-indenter, the CSL-style-rendering phase reports errors. Other PRs that failed because of the re-indenter have been restarted and run green again.

**Comment by retorquere** [comment]
Although we could make it so that if it's labeled safe to test Sheldon will allow mixed patches.

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**